### PR TITLE
Add startup and enabled options

### PR DESCRIPTION
**Pull request by artem-russkikh**
Help to resolve https://github.com/moove-it/sidekiq-scheduler/issues/41
Needs to rewrite README

### CODE REVIEW

File: lib/sidekiq-scheduler/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module SidekiqScheduler
-  VERSION = '1.0'
+  VERSION = '1.1'
 end
```
Comment (elpic) [inline lib/sidekiq-scheduler/version.rb]: Could you please remove the version bump, It's already bumped.

File: test/lib/sidekiq/scheduler_test.rb [hunk 2]
```diff
@@ -4,6 +4,8 @@ class ManagerTest < Minitest::Test
   describe 'Sidekiq::Scheduler' do
 
     before do
+      Sidekiq::Scheduler.startup = true
```
Comment (elpic) [inline test/lib/sidekiq/scheduler_test.rb]: Could you please add more tests around the new options?

**Comment by elpic** [comment]
Please squash all the commits into one.

**Comment by artem-russkikh** [comment]
@elpic I remove 'startup' option, because it is enough 'enabled' option for resolving #41 

'enabled' option is covered by one spec, what kind of test can i write more?

**Pull request merged**